### PR TITLE
Initial commit of draft for Getting Started page refine

### PR DESCRIPTION
**Pull request by carlosmmatos**

### CODE REVIEW

File: assets/src/app/ato/GettingStarted.tsx [hunk 1]
```diff
@@ -11,20 +12,37 @@ const GettingStarted: React.FunctionComponent<any> = (props) => {
...
                   <Gallery gutter="md">
                       <GalleryItem>
                           <Text component="h3">Auditors</Text>
                           <Text component="p">
-                              Content to assist with system accreditation based on the NIST Risk Management Framework. Materials include Security Requirement Traceability Matrixes, product certification materials (Common Criteria, FIPS), and other ATO package documents.
+                                Content to assist with system accreditation based on the NIST Risk Management Framework. Materials include Security Requirement Traceability Matrixes, product certification materials (Common Criteria, FIPS), and other ATO package documents.
                           </Text>
                       </GalleryItem>
                       <GalleryItem>
                           <Text component="h3">Administrators</Text>
```
Comment (redhatrises) [inline assets/src/app/ato/GettingStarted.tsx:31]:
```suggestion
                          <Text component="h3">System Administrators</Text>
```

File: assets/src/app/ato/GettingStarted.tsx [hunk 2]
```diff
@@ -11,20 +12,37 @@ const GettingStarted: React.FunctionComponent<any> = (props) => {
...
+                      <GalleryItem>
+                          <Text component="h3">Security Professionals</Text>
+                          <Text component="p">
+                                You are a security specialist that helps define security requirements and has a deep understanding of how to analyze systems for security threats and vulnerabilities.
+                          </Text>
+                      </GalleryItem>
+                      <GalleryItem>
+                          <Text component="h3">Developers/Application Administrators</Text>
+                          <Text component="p">
+                                You are a developer that builds software and systems that must be hardened to a security compliance baseline. Your goal is to understand early in the process what the security engineering you will need to implement to be compliant.
```
Comment (redhatrises) [inline assets/src/app/ato/GettingStarted.tsx:45]: What about application administrators? Only kinda talking about developers here.

File: assets/src/app/ato/GettingStarted.tsx [hunk 3]
```diff
@@ -11,20 +12,37 @@ const GettingStarted: React.FunctionComponent<any> = (props) => {
               <TextContent>
                   <Text component="h1">Getting Started</Text>
                   <Text component="p">
-                      The Red Hat ATO Pathways microsite provides resources to accelerate your ATO process.
+                    The Red Hat ATO Pathways microsite provides resources to help accelerate your ATO process.
```
Comment (redhatrises) [inline assets/src/app/ato/GettingStarted.tsx:15]: What is ATO? Should define the acronym.

File: assets/src/app/ato/GettingStarted.tsx [hunk 4]
```diff
@@ -11,20 +12,37 @@ const GettingStarted: React.FunctionComponent<any> = (props) => {
...
                   </Text>
+                    <Text component="h2">What Exactly is an ATO?</Text>
+                    <Text component="p">
+                    It's your agency authority certifying your system is ready for day 2: system availability, routine maintenance, and monitoring, and lots of other concerns.
```
Comment (redhatrises) [inline assets/src/app/ato/GettingStarted.tsx:19]: I would probably remove "day 2" and change it to "operations". Reason being is that in some cases it is pre-day 1 or day 1.

File: assets/src/app/ato/GettingStarted.tsx [hunk 5]
```diff
@@ -11,20 +12,37 @@ const GettingStarted: React.FunctionComponent<any> = (props) => {
...
+                  </Text>
+                <Brand src={pic} alt="testing" />
                   <Text component="h2">Role-based guides</Text>
                   <Gallery gutter="md">
                       <GalleryItem>
                           <Text component="h3">Auditors</Text>
                           <Text component="p">
-                              Content to assist with system accreditation based on the NIST Risk Management Framework. Materials include Security Requirement Traceability Matrixes, product certification materials (Common Criteria, FIPS), and other ATO package documents.
+                                Content to assist with system accreditation based on the NIST Risk Management Framework. Materials include Security Requirement Traceability Matrixes, product certification materials (Common Criteria, FIPS), and other ATO package documents.
```
Comment (redhatrises) [inline assets/src/app/ato/GettingStarted.tsx:27]: Probably should define NIST

**Comment by carlosmmatos** [comment]
@redhatrises this page is currently being re-done. For example, I will be getting rid of the Role Based Guides stuff as we don't have these Roles 1) clearly defined and 2) no content behind them. As for the other recommendation,  I will take those into account.

File: assets/src/app/ato/GettingStarted.tsx (hunk 4 above)
Comment (carlosmmatos) [reply assets/src/app/ato/GettingStarted.tsx:19]: We are redefining this whole section

File: assets/src/app/ato/GettingStarted.tsx (hunk 5 above)
Comment (carlosmmatos) [reply assets/src/app/ato/GettingStarted.tsx:27]: This section will be removed for now

File: assets/src/app/ato/GettingStarted.tsx (hunk 3 above)
Comment (carlosmmatos) [reply assets/src/app/ato/GettingStarted.tsx:15]: Will be redefined

**Comment by carlosmmatos** [comment]
I've made significant changes to this page as far as content and layout. This is much better than our existing page so please review it and let's get this thing merged

**Comment by carlosmmatos** [comment]
image
Here's a preview

**Comment by isimluk** [comment]
Wow! This looks fantastic.

**Pull request merged**